docs/articles/documentation/using-testcafe/installing-testcafe.md

```diff
@@ -50,3 +57,53 @@ You can use the local version of TestCafe from a [command line](command-line-int
     ```bash
     testcafe chrome tests/
     ```
+
+## Using TestCafe Docker Image
+
+TestCafe provides a preconfigured Docker image with TestCafe, Chromium and Firefox installed,
+so that you can avoid the need to install browsers or the testing framework on the server.
```

Comment:
`therefore you can avoid manual installation of browsers and the testing framework on the server`